app/controllers/search_controller.rb

```diff
@@ -0,0 +1,32 @@
+class SearchController < ApplicationController
+  def search
+    @free_games = FreeGame.all
+    @giveaways = Giveaway.all
+    @cheap_games = CheapGame.none
+
+    if params[:search].present?
+      search_query = "%#{params[:search]}%"
+      @free_games = @free_games.where("title LIKE ?", search_query)
+      @giveaways = @giveaways.where("title LIKE ?", search_query)
+    end
+
+    if params[:platform].present? && params[:platform] != "All Platforms"
+      if params[:platform] == "PC"
+        @free_games = @free_games.where("platform LIKE ? OR platform LIKE ?", "%PC (Windows)%",
```

Comment:
For these lines or any other pertaining lines when using the LIKE statement we can simply use %PC%

instead of %PC (Windows)%

Because this essentially its making the wildcard search for anything before and after "PC (Windows)"

Realistically if you did %PC% it will search for anything containing the characters "PC"